governance/GOVERNANCE.md

```diff
@@ -57,17 +57,21 @@ To improve organizational transparency and collaboration between the CPC and Boa
 
 ## Consensus Seeking Process
 
-The Team follows a [Consensus Seeking][] decision-making model.
-
-When an agenda item has appeared to reach a consensus, the moderator
-will ask "Does anyone object?" as a final call for dissent from the
-consensus.
-
-If an agenda item cannot reach a consensus, a Team member can call for
-the item to be decided by a vote or to table the issue to the next
-meeting. In both cases the decision must be seconded by a majority of the Team
-or else the discussion will continue. Simple majority wins. Only Active
-Members participate in a vote.
+The Team follows a [Lazy Consensus Seeking][] decision-making model.
+
+Lazy consensus is achieved by making a proposal in a forum that will reach all
+members of the group who should have the opportunity to participate in reaching
+consensus and waiting for an appropriate amount of time for anyone to object.
+It is not necessary to get explicit approval to proceed from all members,
+silence indicates consent. However, objections must be resolved in order
+to achieve consensus.
+
+If a proposal cannot reach a consensus, a group member can call for
+the proposal to be decided by a vote. This should only be done as a last resort
+when consensus cannot be reached after a reasonable period of discussion.
+For all votes, a simple majority of all group voting members for, or against,
+the issue wins. A group voting member may choose to participate in any vote
+through abstention.
```

Comment:
> "A group voting member may choose to participate in any vote
through abstention."

I'm not exactly sure what this means.

Reply:
To me it means that you can participate without saying yes or no, by abstaining.

Reply:
So that's the language that's also in the charter. It's either too explicit or not enough. For example, 

- is the vote of present members or all members?
- is the vote async or not?
- if the vote is of members present, do we need a quorum to have a vote?
- do votes need to be announced in advance?
- is simple majority "more than 50%" or "the answer with a bigger score"?
- if majority is "more than 50%" are abstentions counted?
- if so, what prevents reframing the question as its negative to flip the result of the vote, e.g. 4 people vote for prop `x`, 3 against, and 3 abstain. The vote fails with only 40% of the votes. Now organize a vote for `!x`. You'll get the 4 people that were for `x` vote against `!x`, the 3 that were against `x` now vote for `!x`, and the same 3 as before will abstain. So now we have only 30% for `!x` which fails, an do so `x` passes.
- can anyone request a vote at anytime? If so, that goes against the idea of being consensus-driven. If not, the language needs to be mores specific about when it is OK to ask for a vote.

I don't want us to spend too much time discussing voting as the case never really showed up, so my suggestion would be to acknowledge that voting should be a last resort solution, costly in social capital, painful to organize, hard to weaponize, and favoring the status quo.

I'd suggest the following language:

> Voting should only be done as a last resort. If a proposal to modify existing policies and processes (or add new ones) cannot reach consensus after a reasonable period of discussion, a voting member can call for objections to be overruled through an asynchronous vote. If this call is seconded by another voting member, an asynchronous vote must be organized. For the objections to be overruled (and thus the proposal to pass), at least 2/3 of all voting members must vote in favor. This is designed to make it difficult to change existing policies and processes (or to add new ones) without consensus, but prevent someone from completely blocking progress through objections.

WDYT?

Reply:
> WDYT?

I think the suggestion is a great addition.

Reply:
I also think this is a great addition/clarification of the consensus process for decision making. 
It's a nit and not needed to land these changes, but depending on who is looking at the governance doc and decision making processes, it could help to explicitly state that these processes are applicable to anything in scope for the CPC (defined in the README.md). Nice work!

Reply:
Technically, the scope of the CPC is defined in the charter itself.